controllers/nginx/pkg/template/configmap_test.go

```diff
@@ -53,6 +54,9 @@ func TestMergeConfigMapToStruct(t *testing.T) {
 	def.UseProxyProtocol = true
 	def.GzipTypes = "text/html"
 
+	h, _ := dns.GetSystemNameServers()
+	def.Resolver = h
```

Comment:
Ditto - presumably only if there was not an error?  Although in this case I suspect we fail the test (t.Fatalf) and see what happens...